directories/src/redux/api/peer.js

```diff
@@ -0,0 +1,26 @@
+import { Observable } from 'rxjs';
+import Peer from 'peerjs';
+
+import {
+  PEER_HOST,
+  PEER_PORT,
+  PEER_PATH,
+  PEER_DEBUG
+} from '../../config/';
+
+export default peer =>
+  Observable.create((observer) => {
```

Comment:
let's keep the `Observable` stuff only in the epics

Reply:
what you can do is wrap it in a `Promise` if you want to chain it.

Then you can do
`Observable.fromPromise(connectPeer()).map(peer => myAction.peerConnected(peer))` from one of the epics